meilisearch-http/src/option.rs

```diff
@@ -93,6 +93,26 @@ pub struct Opt {
     /// SSL support tickets.
     #[structopt(long, env = "MEILI_SSL_TICKETS")]
     pub ssl_tickets: bool,
+
+    /// Defines the path of the snapshot file to import.
+    /// This command will stop the process if a database already exist or if no snapshot exists in the given path.
+    /// If this command is not called, no snapshot will be imported.
+    #[structopt(long, env = "MEILI_INIT_FROM_SNAPSHOT")]
+    pub init_from_snapshot: Option<PathBuf>,
+
+    /// Defines the path of the snapshot file to import.
+    /// This command will skip importing if a database already exist or if no snapshot exists in the given path.
+    /// If this command is not called, no snapshot will be imported.
+    #[structopt(long, conflicts_with = "init-from-snapshot", env = "MEILI_RECOVER_FROM_SNAPSHOT")]
+    pub recover_from_snapshot: Option<PathBuf>,
```

Comment:
I would like to replace both the `init-from-snapshot` and `recover-from-snapshot` by one `load-from-snapshot` option that can be associated with an `ignore-missing-snapshot` option. This way we avoid potential confusions and keep the documentation of the snapshot loading option in one place and specify snapshot loading behaviors with simple bolean flags.

I don't know what to do when a database already exists, maybe never return an error and just thrown a warning?
I'm not sure it would be a good idea to introduce another flag? What do you think?

```suggestion
    /// Defines the path of the snapshot file to import.
    /// This option will, by default, stop the process if a database already exist or if no snapshot exists at
    /// the given path. If this option is not specified no snapshot is imported.
    #[structopt(long, env = "MEILI_LOAD_FROM_SNAPSHOT")]
    pub load_from_snapshot: Option<PathBuf>,

    /// The engine will ignore a missing snapshot and not return an error in such case.
    #[structopt(long, requires = "load-from-snapshot", env = "MEILI_IGNORE_MISSING_SNAPSHOT")]
    pub ignore_missing_snapshot: bool,
```

Reply:
We concluded in using the `ignore-snapshot-if-db-exists` to make the engine throw an error if the database already exists.

Reply:
so the default behavior is to erase user data? Isn't it good practice to do the other way around? (`rm -f`, `git push -f` etc...)

Reply:
@MarinPostma nope, db will never be erased, the behavior of `ignore-snapshot-if-db-exists` is:
* `false` (default): crash if db exists
* `true`: skip snapshoting process and run meilisearch with actual db